README.md

```diff
@@ -380,7 +380,7 @@ Futher detail [here (staff only)](https://docs.google.com/document/d/1muN4pFqRhL
   <tr>
     <td>
       <ul>
-        <li>Plays a strategic role in the stream’s technical decision making through data-driven decision-making whenever possible.</li>
+        <li>Plays a strategic role in the stream’s technical decision making through data-driven decision-making whenever possible†.</li>
```

Comment:
where relevant? I prefer data-informed rather than data-driven tbh as a product development philosophy, but not sure if that's guardian-wide!

Reply:
amended, thanks!